homeassistant/components/ruuvitag_ble/config_flow.py

```diff
@@ -0,0 +1,93 @@
+"""Config flow for ruuvitag_ble."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+from ruuvitag_ble import RuuvitagBluetoothDeviceData
+
+from homeassistant.components.bluetooth import (
+    BluetoothServiceInfoBleak,
+    async_discovered_service_info,
+)
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.data_entry_flow import FlowResult
+from .const import DOMAIN
+
+
```

Comment:
We do need tests for the config flow, but you can likely copy them from another one of the ble integrations

Reply:
It's exactly the same config flow as `tilt_ble` with `TiltBluetoothDeviceData` having been replaced with `RuuvitagBluetoothDeviceData`...